test/group-3.jl

```diff
@@ -132,13 +134,14 @@ end
     @test Tullio.onlyone(true, 0) === true
     @test Tullio.onlyone(true, 1) === false
 
-    # @test Tullio.onlyone(ms, 0) === Mask{UInt8}(0x02)
-    @test Tullio.onlyone(ms, 0).u == 0x02
-    # @test Tullio.onlyone(ms, sv) === Mask{UInt8}(0x00)
-    @test Tullio.onlyone(ms, sv).u == 0x00
-    # @test Tullio.onlyone(ms, zero(sv)) === Mask{UInt8}(0x02)
-    @test Tullio.onlyone(ms, zero(sv)).u == 0x02
+    @test Tullio.onlyone(ms, 0) === Mask{4}(0x02)
+    # @test Tullio.onlyone(ms, 0).u == 0x02
+    @test Tullio.onlyone(ms, sv) === Mask{4}(0x00)
+    # @test Tullio.onlyone(ms, sv).u == 0x00
+    @test Tullio.onlyone(ms, zero(sv)) === Mask{4}(0x02)
+    # @test Tullio.onlyone(ms, zero(sv)).u == 0x02
 end
+=#
 
 @testset "parsing + LoopVectorization" begin include("parsing.jl") end
 
```

Comment:
```suggestion

Threads.nthreads() > 1 && exit()  # Github CI times out on multi-thread LV gradient checks.

```
Like so, after 1h 3m 48s which I presume means the robot-slayer checks the clock every 5 minutes:
```
130
Test Summary:               | Pass  Broken  Total
131
parsing + LoopVectorization |  244       4    248
132
ERROR: Package Tullio errored during testing (received signal: KILL)
133
Stacktrace:
134
 [1] pkgerror(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Types.jl:52
135
 ```